content/en/docs/instrumentation/go/resources.md

```diff
@@ -44,7 +45,7 @@ resources := resource.New(context.Background(),
     resource.WithOS(), // This option configures a set of Detectors that discover OS information
     resource.WithContainer(), // This option configures a set of Detectors that discover container information
     resource.WithHost(), // This option configures a set of Detectors that discover host information
-    resource.WithDetectors(thirdparty.Detector{}), // Bring your own external Detector implementation
+    resource.WithDetectors(thirdParty.Detector{}), // Bring your own external Detector implementation
```

Comment:
```suggestion
    resource.WithDetectors(thirdparty.Detector{}), // Bring your own external Detector implementation
```

`thirdparty` is a name of the Go package name. The idiomatic way is to have them lowercase. See https://go.dev/blog/package-names:

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.

Reply:
My assumption was that this is a place holder for any kind of third party package, isn't it?

Reply:
True. But even a placeholder should use idiomatic naming to look good 😉